test/setup.mjs

```diff
@@ -0,0 +1,3 @@
+register("some-typescript-loader");
```

Comment:
Whats the point of this?

Reply:
I thought it was necessary because it was in the document, but we don't need it, I think I take it back